### PR TITLE
Removed important changed type of lambda

### PR DESCRIPTION
**Pull request by KingRealzYT**
Removed important changed type of lambda

### CODE REVIEW

File: src/main/java/com/crypticcosmos/crypticcosmos/registries/StrippingRegistries.java [hunk 1]
```diff
@@ -45,9 +43,7 @@ public static void onBlockClicked(PlayerInteractEvent.RightClickBlock event) {
                     world.setBlock(blockPos, block.defaultBlockState()
                             .setValue(RotatedPillarBlock.AXIS, blockState.getValue(RotatedPillarBlock.AXIS)), 11);
                     if (playerEntity != null) {
-                        event.getItemStack().hurtAndBreak(1, playerEntity, (p_220040_1_) -> {
-                            p_220040_1_.broadcastBreakEvent(event.getHand());
-                        });
+                        event.getItemStack().hurtAndBreak(1, playerEntity, (p_220040_1_) -> p_220040_1_.broadcastBreakEvent(event.getHand()));
```
Comment (TheOnlyTails) [inline src/main/java/com/crypticcosmos/crypticcosmos/registries/StrippingRegistries.java:46]: Can you please use something other than the unmapped name and remove the unnecessary parentheses?

**Review by TheOnlyTails — CHANGES_REQUESTED** [review]
LGTM, just change the parameter name.

File: src/main/java/com/crypticcosmos/crypticcosmos/registries/StrippingRegistries.java (hunk 1 above)
Comment (KingRealzYT) [reply src/main/java/com/crypticcosmos/crypticcosmos/registries/StrippingRegistries.java:46]: yea sure

**Review by TheOnlyTails — APPROVED** [review]
LGTM!

**Pull request merged**